style.css

```diff
@@ -330,3 +334,15 @@ footer::before {
     height: 180px;
   }
 }
+
+@media screen and (max-width: 650px) {
+  .modal-container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media screen and (max-width: 500px) {
+  .modal-container {
+    grid-template-columns: repeat(1, 1fr);
```

Comment:
Hey, Thomas, I think repeat is unnecessary, do you mind to remove it.

Reply:
I will update it @selftechacademy